man/loader.conf.xml

```diff
@@ -399,6 +399,17 @@ sbvarsign --attr "${attr}" --key KEK.key --cert KEK.pem --output db.auth db db.e
 
         <xi:include href="version-info.xml" xpointer="v251"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term>group-entries</term>
+
+        <listitem><para>Takes a boolean argument or <literal>auto</literal>. If true, menu entries are
+        grouped by OS. If false, all menu entries are displayed sequentially. If set to
+        <literal>auto</literal> (the default), menu entries are only grouped if multiple OSes are installed.
+        </para>
+
+        <xi:include href="version-info.xml" xpointer="v256"/></listitem>
+      </varlistentry>
```

Comment:
why make this configurable? I don't like unnecessary configuration...

Reply:
and i think we should do this grouping only if the menu grows too large if it's maintained as a plain list. i.e. add a check if number of entries is < 10 or so stick to the simple menu, otherwise see of collapsing entries might shorten things.